src/exchange/information/information.html

```diff
@@ -4,257 +4,205 @@
-                                </li>
-                                <li class="oui-tile__item"
-                                    data-ng-if="ctrl.exchange.sslExpirationDate">
-                                    <div class="oui-tile__definition">
-                                        <div class="d-flex w-100">
-                                            <div>
-                                                <strong class="oui-tile__term d-block"
-                                                        data-translate="exchange_dashboard_ssl_expiration"></strong>
-                                                <span data-ng-bind="ctrl.exchange.sslExpirationDate | date:'mediumDate'"></span>
-                                            </div>
-                                        </div>
-                                        <div class="alert alert-info mt-3 mb-0" id="sslRenewal" role="alert"
-                                             data-ng-bind="ctrl.messageSSL"
-                                             data-ng-if="ctrl.messageSSL">
-                                        </div>
-                                    </div>
-                                    <oui-action-menu data-compact 
-                                                     data-align="end"
-                                                     data-disabled="ctrl.hasSSLTask"
-                                                     data-ng-if="ctrl.shouldDisplaySSLRenewValue">
-                                        <oui-action-menu-item data-on-click="ctrl.sslRenew()">
-                                            <span data-translate="exchange_action_renew_ssl"></span>     
-                                        </oui-action-menu-item>
-                                    </oui-action-menu>
-                                </li>
-                                <li class="oui-tile__item"
-                                    data-ng-if="ctrl.displayRenewDate()">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block"
-                                                data-translate="exchange_dashboard_expiration"></strong>
-                                        <span data-ng-bind="ctrl.exchange.expiration | date:'mediumDate'"></span>
-                                    </div>
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'exchange_tab_INFORMATIONS_summary' | translate}}">
+                        <oui-tile-definition data-term="{{:: 'exchange_tab_INFORMATIONS_infrastructure' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="(('exchange_offer_type_' + ctrl.exchange.offer) | translate) || ctrl.exchange.offer"></span>
```

Comment:
This ng-bind could be done directly into a data-description prop of oui-tile-definition tag (as following) : 

```suggestion
<oui-tile-definition data-term="{{:: 'exchange_tab_INFORMATIONS_infrastructure' | translate}}" 
                             data-description="{{ (('exchange_offer_type_' + ctrl.exchange.offer) | translate) || ctrl.exchange.offer }}">
```

Reply:
Fixed for all case 😄